### DISCUSSION TITLE
Allow larger requests: Unable to post/patch content in excess of 100kb

### DISCUSSION THREAD

**Discussion by sessionsdev** [coddingtonbear/obsidian-local-rest-api#73] [Ideas]
Express JS has a default limit of 100kb for payload size.  Can this be increased or made configurable by the user?

10mb is quite reasonable for this kind of app, imo.

**Comment by sessionsdev** [comment]
I tried forking and tested some things myself.  This might be a limitation of the obsidian api.  Close if the LoE is too high here.

**Comment by coddingtonbear** [comment]
Neat -- I hadn't considered that ExpressJS might have a limit, but.  I'm not sure, @sessionsdev,  if you've tried making changes here: https://github.com/coddingtonbear/obsidian-local-rest-api/blob/main/src/requestHandler.ts#L51, but I'd be willing to review and merge a PR if you could make the fix increasing it to ~10MB.

**Comment by fuergaosi233** [comment]
Same problem

**Reply by coddingtonbear** [reply]
For sure -- feel free to see if you can find out how to increase the limit and post a pull request, @fuergaosi233.  I won't be finding the time to sort this out myself since this isn't something that overlaps with my use-case, but if you can find the problem and fix it, it could solve it for both yours and those of the others in this thread.

**Comment by coddingtonbear** [comment]
I just happened across the fix for this while working through a pull request review and increased the maximum incoming payload size to 1024MB (see 22cdd4229acd043765f7ddc30972f2097d5f400c).
